### ISSUE TITLE
Problem 1, Hill 2: Idea to improve accountability and awareness when responding to a situation

### ISSUE THREAD

**Issue by jharvey10** [Call-for-Code/Embrace-Judicial-Reform#2]
### Theme
Problem Statement 1, Hill 2
> Law enforcement has technology embedded in their equipment to help objectively assess accurate threat level and provide guidance on appropriate threat response in real time.

### Brief description of your idea
Use AI/machine learning/analytics to pre-assess the threat level of a situation prior to officers responding to an incident. The goal is to increase accountability, scrutiny, and consequence through knowledge and analytics to reduce conflict escalation as much as possible.

The foundation of this idea is this equation:

`low threat` + `high use of force` = `high level of scrutiny`

The proposed idea would provide the `[low|medium|high] threat` part of this equation and would rely on data and reporting from solutions such as #1 to determine the level of force that was used and how this relates to the expected outcome of the situation. These two factors would then dictate an appropriate level of scrutiny, accountability, and consequence.

### Some details
When officers respond to incidents, they are undoubtedly given some information prior to arriving, but I am not sure what this set of information includes. I believe that this set of information could be changed based on research, analytics, and real-time data to give citizens and officers the best possible chance of low-escalation conflict resolution.

The key would be to come up with an "estimated threat level" of a given situation/incident based on AI, machine learning, and analytics. This threat level and other relevant details will be relayed to the officer and must be acknowledged prior to arriving at the scene. Based on the threat level, it will be established that certain uses of force or weaponry will be scrutinized much more heavily in low-severity situations than in those with significantly higher estimated threat levels. The model should use concrete facts and data that factor *out* racial bias as a basis for threat severity.

### What makes your idea unique?
I believe this idea is unique because it represent a proactive approach. The key to this implementation is determining which factors are most useful in generating an estimated threat level, and communicating this in an effective way to responding officers.

The threat modeling could pull data from a number of sources, depending on which result in the best trained predictive models. Some examples of data sources might be:

- Historical data for relevant incident types
- Historical crime data for relevant area
- Typical tactics used in these types of incidents
- Best options for de-escalation for relevant incident types
- Time of day
- Visibility/available light/weather
- How busy an area currently is (i.e. how many people are around)
- What events are going on in the area (public social media info)
- Officer's estimated stress level based on driving characteristics or heat rate data
- Number of responding officers
- Data from bystanders (though this has a large potential for any number of biases)
- etc...

### What would be the impact of your idea if implemented?
The main impacts are accountability and awareness. Awareness arms officers with *knowledge* in favor of *force* before stepping into a situation. Accountability stresses "real consequence" when deviating from anticipated behavior based on predictive models. 

### Skills to contribute (e.g. development, architecture, research, design or anything else)
This would probably need:
- User Researcher to investigate types of data we can obtain and their levels of success
- Data scientist and architect to help generate and train AI models, and to help provide the correct interfaces into the generated models **I have some minimal understanding of AI/machine learning, but not a lot**
- Developer to create an application law enforcement can use that interfaces with the AI models **I am a developer and could definitely fill this role**
- Designer to create best possible presentation to law enforcement to stress accountability and awareness
- *Possibly others?*

**Comment by daveshack-ibm** [comment]
Things like visibility, light level etc can be dynamically updated as the situation develops right?
Also how about having the officer(s) narrate their thoughts while responding as an added means for accountability and perhaps to detect an impending crisis situation.

**Comment by jharvey10** [comment]
@daveshack-ibm Yeah! absolutely. I know the AI models kind of need to be trained up front and refined over time, but yeah there's definitely two categories of data there, one of which is more of a real-time set of metrics. 

I hadn't put too much thought into what happens *after* the information is relayed to the officer, but I think you point out an important next step (also evident in #1 ), which is to do data collection about each situation early and often.

**Comment by jharvey10** [comment]
Added some additional details to the description for clarity.
